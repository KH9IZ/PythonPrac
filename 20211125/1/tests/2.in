!Knyazev_Egor_Ivanovich_too_much__Knyazev_Egor_Ivanovich_too_much__Knyazev_Egor_Ivanovich_too_much__