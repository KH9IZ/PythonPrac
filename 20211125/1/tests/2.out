!KnyKnyKny_Iv_Iv_Ivanoanoanoazeazeazegorgorgorh__h__h__h_th_th_tmucmucmucoo_oo_oo_v_Ev_Ev_Evicvicvic
